ratchet.txt

```diff
@@ -14,3 +14,6 @@ tests.test_schema.SchemaTest.test_semantic_error
 tests.test_schema.SchemaTest.test_semantic_error_on_value_derived
 tests.test_utils.UtilTest.test_chunks
 tests.test_utils.UtilTest.test_load_file
+tests.test_s3crypt.TestS3CryptEvent.test_s3_event_simple
+tests.test_s3crypt.TestS3CryptEvent.test_s3_event_unsafe_key
```

Comment:
can we just make this py3 compatible, we're trying to draw down on py3 incompatible tests. you'll need to the six moves works to pickup the url encoding lib. six is installed in the lambda env (boto3 has it as a dep).

Reply:
OK